lms/djangoapps/courseware/access.py

```diff
@@ -136,6 +140,36 @@ def can_load_forum():
             )
         )
 
+    def can_load_mobile():
+        """
+        Can this user access this course from a mobile device?
+        """
+        return (
+            # check mobile requirements
+            can_load_mobile_not_enrolled() and
+            # check enrollment
+            (
+                CourseEnrollment.is_enrolled(user, course.id) or
+                _has_staff_access_to_descriptor(user, course, course.id)
```

Comment:
Renamed load_mobile_not_enrolled -> load_mobile_no_enrollment_check in latest commit.